### ISSUE TITLE
Deck templates, badges, guidelines for event promotion

### ISSUE THREAD

**Issue by suxstellino** [sqlcollaborative/DataSaturdays#29]
We should provide a repository with the resources for:

- deck template
- intro/outro for videos of the speaker/event (optional)
- badge "I'll speak at"
- logos
- guidelines for sub brands (optional, if a sub brand will exist
- guidelines for videos (optional)

I've organized and promoted an event with videos and it's been so good marketing side. Just an idea:

- the speaker can promote itself with a 30sec video of its session
- the organizers can promote the event with a video

This is the reason why I would suggest a guideline doc.

**Comment by SQLDBAWithABeard** [comment]
This is an awesome suggestion - How can we add that to this repo for others to use?

**Comment by suxstellino** [comment]
We can create another repo which is called "*-resources". In our community we have:
![Screenshot_2020-12-19-10-26-08-249_com.android.chrome.jpg](https://user-images.githubusercontent.com/2568107/102685972-b03bf380-41e4-11eb-86cd-0b230e9c401a.jpg)

1. For the generated website (Hugo)
2. For the website source (posts/articles)
3. For the assets (resources)

The brand-assets repo is accessible to whomever requires to generate stuff for the site and for videos, like interviews

**Comment by spaghettidba** [comment]
I like the idea, but would somehow prefer to have everything in a single repo. Do you think that's possible?

**Comment by gavincampbell** [comment]
Weren't deck templates a big source of complaints about the PASS setup?

**Comment by suxstellino** [comment]
> I like the idea, but would somehow prefer to have everything in a single repo. Do you think that's possible?

Yes, up to us 😉. It could be just a subfolder with its own wiki/readme.

**Comment by suxstellino** [comment]
> Weren't deck templates a big source of complaints about the PASS setup?

If you design a good template to work with (i.e. more than one resolution supported, good color balances, just opening and closing slides, and so on) I think it could be useful. We can store any template, like one per edition/city also.

**Comment by amtwo** [comment]
I just submitted a PR (#44) to create a directory within this repo for branding/marketing bits, and included the start of a "brand book" that we can use to list what colors, fonts, and logos to use, and when. Essentially--a thing so I don't have to parse CSS to find these details from the website. 😁

I'm going to propose that this gets split into a series of smaller issues. Each of these marketing/branding items is really a separate item to tackle--admittedly with interdependencies & prerequisites among them. Also, different items come with different priority. Finalizing the base brand identity (Colors, fonts, logo) should probably come early enough to keep other items consistent, rather than everyone going their own way & having to update 1000 small items later.

**Comment by spaghettidba** [comment]
I literally grabbed a semi-random bootstrap template from a template gallery and generated a logo with a logo generator. This should not be taken as a complete or definitive design. I suck at design
